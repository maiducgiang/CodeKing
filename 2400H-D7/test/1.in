7
32 37
23 13
30 13
4 19
18 49
21 44
19 26
