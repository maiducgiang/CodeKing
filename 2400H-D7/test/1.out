613493884
67852481
510206587
530096591
612702561
486724501
367532587
